packages/calypso-e2e/src/browser-helper.ts

```diff
@@ -1,55 +1,59 @@
+/**
+ * External dependencies
+ */
+import config from 'config';
+
 /**
  * Internal dependencies
  */
-import type { screenSize, localeCode } from './types';
+import type { viewportName, localeCode, viewportSize } from './types';
 
 /**
  * Returns the target screen size for tests to run against.
  *
- * By default, this function will return 'desktop' as the target.
- * To specify another screen size, set the BROWSERSIZE environment variable.
+ * If the environment variable BROWSERSIZE is set, this will override all configuration
+ * values. Otherwise, the default path contained in the configuration file is returned.
  *
- * @returns {screenSize} Target screen size.
+ * @returns {viewportName} Target screen size.
  */
-export function getTargetScreenSize(): screenSize {
-	return ! process.env.BROWSERSIZE
-		? 'desktop'
-		: ( process.env.BROWSERSIZE.toLowerCase() as screenSize );
+export function getViewportName(): viewportName {
+	return ! process.env.VIEWPORT_NAME
+		? config.get( 'viewportName' )
+		: ( process.env.VIEWPORT_NAME.toLowerCase() as viewportName );
```

Comment:
Would it be better to call `toLowerCase` on the output value, regardless where it's taken from? Or does TS validate the config values? (sorry, I don't have much exp. here). I'm thinking something like
```js
return ( process.env.VIEWPORT_NAME || config.get( 'viewportName' ) ).toLowerCase();
```

Reply:
You're right, it's possible for someone to specify the `viewportName` in uppercase or camelcase. I thought the `viewportName` return type __should__ catch any instances of that, but let's look at ways to simplify this statement.